packages/react-composites/src/composites/CallComposite/pages/LobbyPage.tsx

```diff
@@ -56,14 +72,31 @@ export const LobbyPage = (props: LobbyPageProps): JSX.Element => {
       mobileView={props.mobileView}
       /* @conditional-compile-remove(one-to-n-calling) */
       modalLayerHostId={props.modalLayerHostId}
-      onRenderGalleryContent={() => <LobbyTile {...lobbyProps} overlayProps={overlayProps(strings, inLobby)} />}
+      onRenderGalleryContent={() => (
+        <LobbyTile {...lobbyProps} overlayProps={overlayProps(strings, inLobby, remoteParticipants)} />
+      )}
       dataUiId={'lobby-page'}
     />
   );
 };
 
-const overlayProps = (strings: CallCompositeStrings, inLobby: boolean): LobbyOverlayProps =>
-  inLobby ? overlayPropsWaitingToBeAdmitted(strings) : overlayPropsConnectingToCall(strings);
+const overlayProps = (
+  strings: CallCompositeStrings,
+  inLobby: boolean,
+  remoteParticipants: RemoteParticipantState[] | undefined
+): LobbyOverlayProps => {
+  /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
```

Comment:
Remove all conditional compilation directives from this function. `remoteParticipants` will always be undefined for stable build.